modules/ghlint-extensions/src/main/kotlin/net/twisterrob/ghlint/model/effectivePermissionsSet.kt

```diff
@@ -0,0 +1,30 @@
+package net.twisterrob.ghlint.model
+
+public fun Permissions.asEffectivePermissionsSet(): Set<Scope> {
```

Comment:
Interesting - happy to make the change, in Swift, conventionally computed variables are O(1).

Reply:
In that case most model properties in ghlint need to be funs :)

I usually go for readability at use-site. The fact that it's calculated every time is an implementation detail, theoretically it could be a `by lazy { }` (if that worked for extension properties).